operator/internal/manifests/internal/config/loki-config.yaml

```diff
@@ -468,9 +478,26 @@ server:
 {{- end }}
   log_level: info
 storage_config:
+{{- range $ship, $_ := $shippersDic }}
+{{- if eq $ship "boltdb" }}
   boltdb_shipper:
-    active_index_directory: {{ .StorageDirectory }}/index
-    cache_location: {{ .StorageDirectory }}/index_cache
+    active_index_directory: {{ $.StorageDirectory }}/index
+    cache_location: {{ $.StorageDirectory }}/index_cache
+{{- end }}
+{{- if eq $ship "tsdb" }}
+  tsdb_shipper:
+    active_index_directory: {{ $.StorageDirectory }}/tsdb-index
+    cache_location: {{ $.StorageDirectory }}/tsdb-cache
+{{- end }}
+{{template "commonStorageConfig" $ }}
+{{- end }}
+tracing:
+  enabled: false
+analytics:
+  reporting_enabled: {{ .EnableRemoteReporting }}
+
+
+{{define "commonStorageConfig" }}
```

Comment:
`commonStorageConfig` only seems to be used once, why is it extracted as a sub-template?